core/src/main/java/org/kohsuke/stapler/RequestImpl.java

```diff
@@ -86,11 +86,11 @@
 import org.kohsuke.stapler.util.IllegalReflectiveAccessLogHandler;
 
 /**
- * {@link StaplerRequest} implementation.
+ * {@link StaplerRequest2} implementation.
  *
  * @author Kohsuke Kawaguchi
  */
-public class RequestImpl extends HttpServletRequestWrapper implements StaplerRequest {
+public class RequestImpl extends HttpServletRequestWrapper implements StaplerRequest2 {
```

Comment:
This implementation class is changing incompatibly to implement `StaplerRequest2`. With the exception of a single proprietary plugin, no plugins care about the implementation class, just the interface.